pyiron_gui/project/project_browser.py

```diff
@@ -249,13 +469,24 @@ def _click_option_button(self, b):
 
     @property
     def data(self):
-        return self._data
+        if self._data is not None:
+            return self._data
+        if isinstance(self.project, PyironWrapper):
```

Comment:
Nit) `if` -> `elif` improves readibilty the same way as earlier, where we have this consecutive clause bunch each wrapping their own `return`. IMO separating the first two `if`s implies some difference in intention, which I don't see here.